extensions/sql-database-projects/src/models/tree/baseTreeItem.ts

```diff
@@ -11,11 +11,11 @@ import * as path from 'path';
  */
 export abstract class BaseProjectTreeItem {
 	/** Project-relative URI that's compatible with the project tree */
-	projectUri: vscode.Uri;
+	relativeProjectUri: vscode.Uri;
 	parent?: BaseProjectTreeItem;
 
-	constructor(uri: vscode.Uri, parent?: BaseProjectTreeItem) {
-		this.projectUri = uri;
+	constructor(relativeProjectUri: vscode.Uri, parent?: BaseProjectTreeItem) {
```

Comment:
I may have still been learning typescript when I wrote this, but `relativeProjectUri` can be changed to be declared inline in the constructor, yeah?